source/upgrade.txt

```diff
@@ -109,13 +109,102 @@ Version 5.0 Breaking Changes
 - This driver version removes the
   ``org.mongodb.scala.ObservableImplicits.ToSingleObservableVoid`` implicit
   class. This means the ``org.reactivestreams.Publisher[Void]`` type no longer
-  converts automatically to ``org.mongodb.scala.SingleObservable[Void]``. The also
-  API exposes ``org.mongodb.scala.Observable[Unit]`` instead of
+  converts automatically to ``org.mongodb.scala.SingleObservable[Void]``. The
+  API also exposes ``org.mongodb.scala.Observable[Unit]`` instead of
   ``org.mongodb.scala.Observable[Void]``. 
   
   .. After the 5.0 Scala API docs are released, this line will be uncommented. 
     For more information, see the `Observable trait documentation <https://mongodb.github.io/mongo-java-driver/5.0/apidocs/mongo-scala-driver/org/mongodb/scala/Observable.html>`__.
 
+- This driver version removes the ``Parameterizable`` interface. Instead of
```

Comment:
Almost all of these entries start with "This driver version removes...". This is clear and accurate, but might be tiresome to read. If you think this info should be organized differently, let me know.

Reply:
Suggestion (not part of a review):

Similar to recent Node.js driver entries, I think these could be made more concise by omitting the implied "This driver version". E.g.
"Removes the ... implicit class".

Another item that I think could be helpful is to mention this is applicable to the Scala driver. Although these What's New entries currently combine all the off-domain Java-related driver updates, this site is mainly for Java Sync. Therefore, it could be helpful to highlight any item that doesn't apply to Java Sync by stating which drivers it impacts.

E.g. Perhaps something like this: 

"Removes the ... implicit class from the Scala driver package."